packages/flutter_driver/test/src/real_tests/flutter_driver_test.dart

```diff
@@ -591,6 +691,95 @@ void main() {
     });
   });
 
+  group('WebFlutterDriver with logCommunicationToFile', () {
+    late FakeFlutterWebConnection fakeConnection;
+    late WebFlutterDriver driver;
+    int driverId = 0;
+
+    setUp(() {
+      _cleanTestOutputFiles();
```

Comment:
What files is this leaving in the output directory? If you're writing out temp stuff, you should create a temp directory instead

Reply:
this code is redundant, i removed.